AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_relocator_tests.py

```diff
@@ -1562,8 +1557,8 @@ def map_env(self, env: dict, source_dir):
         MoveTest(
             description="Move a file: source = absolute-file; destination = absolute (path outside scan folder)",
             asset_folder="one_txt_file",
-            encoded_command="project_test_assets_dir\\testFile.txt,dev_dir\\MoveOutput\\",
-            encoded_output_dir="dev_dir\\MoveOutput\\",
+            encoded_command="project_test_assets_dir\\testFile.txt,C:\\MoveOutput\\",
```

Comment:
will this actually work on Jenkins? Shouldn't this be dev_dir so the test can update it with a safe for Jenkins absolute path, instead of C, which may not exist on Jenkins?